frontend/javascripts/oxalis/view/jobs/train_ai_model.tsx

```diff
@@ -425,19 +468,43 @@ function AnnotationsCsvInput({
           name: annotation.dataSetName,
         });
 
-        const volumeTracings: ServerVolumeTracing[] = await Promise.all(
+        const volumeServerTracings: ServerVolumeTracing[] = await Promise.all(
           annotation.annotationLayers
-            .filter((layer) => layer.typ === "Skeleton")
+            .filter((layer) => layer.typ === "Volume")
             .map(
               (layer) =>
                 getTracingForAnnotationType(annotation, layer) as Promise<ServerVolumeTracing>,
             ),
         );
+        // TODO: make bboxs a member again
+        const volumeTracings = volumeServerTracings.map((tracing) =>
+          serverVolumeToClientVolumeTracing(tracing),
+        );
+        let userBoundingBoxes = volumeTracings[0]?.userBoundingBoxes;
+        if (!userBoundingBoxes) {
+          const skeletonLayer = annotation.annotationLayers.find(
+            (layer) => layer.typ === "Skeleton",
+          );
+          if (skeletonLayer) {
+            const skeletonTracing = await getTracingForAnnotationType(annotation, skeletonLayer);
+            userBoundingBoxes = convertUserBoundingBoxesFromServerToFrontend(
+              skeletonTracing.userBoundingBoxes,
+            );
+          } else {
+            new Error(`Annotation ${annotation.id} has neither a volume nor a skeleton layer`);
```

Comment:
Should this error be thrown? Otherwise, I think the line doesn't have any effect :thinking:

Reply:
Yeah, I wanted to throw an error here in case somehow no tracing exists for an annotation (which should never be the case). By throwing an error I was hoping to show the user a useful error message. But as this should not happen per se, I did not make an effort to show a toast and log an error and so on and simply wanted to rely on the default error catching mechanism of wk.

Reply:
Yes, but I meant that this is missing the actual `throw`, doesn't it? :)

```suggestion
            throw new Error(`Annotation ${annotation.id} has neither a volume nor a skeleton layer`);
```